tests/OTAnalytics/plugin_prototypes/track_visualization/test_track_viz.py

```diff
@@ -81,20 +82,37 @@ def test_get_data_empty_track_repository(self) -> None:
 
 
 class TestCachedPandasTrackProvider:
-    def test_get_data_reset_cache(
-        self,
-    ) -> None:
-        id = TrackId(1)
+    def create_tracks(self) -> None:
```

Comment:
We could replace this method with pytest fixtures for each track created here.
Pytest will provide them via dependency injection when declaring them as parameters in the test method.

```python
    @pytest.fixture
    def track_1(self) -> Track:
        return self.set_up_track(1)

    @pytest.fixture
    def track_2(self) -> Track:
        return self.set_up_track(2)
    
    def test_foo(self, track_1: Track) -> None:
        assert 1 == track_1.id.id